packages/radio-box-group/src/RadioBox.tsx

```diff
@@ -233,9 +237,14 @@ export default function RadioBox({
     showFocus,
   });
 
+  const tooltipTriggerHandlers = getTooltipTriggerHandlers<HTMLLabelElement>(
+    rest as HTMLProps<HTMLLabelElement>,
+  );
+
   return (
     <label
       {...radioBoxWrapper.prop}
```

Comment:
Without having looked at the rest of the code, I'm confused why this says `...radioBoxWrapper.prop`. Why is it not either `props` or a more descriptive name?

Reply:
`radioBoxWrapper` is the object returned from `createDataProp`